www/master/components/dashboard/test/controllers/header.spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+describe("header controller", function() {
```

Comment:
Can you add a comment or something explaining what this tests, for those of us who find js mystifying?